src/Network/Ethereum/Web3/Types/Types.purs

```diff
@@ -282,7 +319,6 @@ newtype TransactionOptions =
                      }
 
 derive instance genericTransactionOptions :: Generic TransactionOptions _
-derive instance newtypeTransactionOptions :: Newtype TransactionOptions _
```

Comment:
Is this intentionally removed? Is the idea to use lenses in general? (Note: I like lenses but without template purescript or some automatic method of derivation they're a bit annoying)

Reply:
no this is not intentional